README.md

```diff
@@ -30,26 +30,17 @@ for example code.
 
 ## Local Development
 
-### `npm start`
+### `npm run start -- BACKEND NETWORK CONFIG-DIR STATE-DIR`
 
-Runs the project in development/watch mode.
+Run the CLI during development, reloading on changes.
 
 ### `npm run build`
 
-Bundles the package to the `dist` folder.
+TS -> JS in `dist`
 
-### `npm test`
+### `npm test` `npm test -- --watch unit`
 
-Runs the test watcher ([Jest](https://jestjs.io/docs/en/getting-started))
-in an interactive mode.
-By default, runs tests related to files changed since the last commit. 
-
-Alternatively, to run only unit tests:
-
-    npx tsdx test unit
-
-See the [Jest command-line reference](https://jestjs.io/docs/en/cli)
-for all the options.
+See [Jest CLI Options](https://jestjs.io/docs/en/cli.html) 
```

Comment:
It's helpful to provide more information about running tests because this will be the first time many devs have even heard of Jest.
 
```suggestion
Runs all tests.

To run only `unit` tests matching `idempotent` (for example):

    npm test unit -- -t idempotent

Run the test watcher ([Jest](https://jestjs.io/docs/en/getting-started)) and prompt for usage:

    npm test -- --watch

See the [Jest command-line reference](https://jestjs.io/docs/en/cli) for all the options.
```